homeassistant/components/peco/sensor.py

```diff
@@ -58,27 +64,33 @@ class PecoOutageCounterSensorEntity(CoordinatorEntity, SensorEntity):
     """PECO outage counter sensor."""
 
     _attr_state_class = SensorStateClass.MEASUREMENT
-    _attr_icon = "mdi:power-plug-off"
-
-    def __init__(self, hass, name, county, coordinator):
+    _attr_icon: str = "mdi:power-plug-off"
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        name: str,
+        county: str,
+        coordinator: DataUpdateCoordinator,
+    ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
-        self.hass = hass
-        self._county = county
-        self._name = name
+        self.hass: HomeAssistant = hass
+        self._county: str = county
+        self._name: str = name
 
     @property
-    def name(self):
+    def name(self) -> str:
```

Comment:
consider using `_attr_` variables instead of properties where possible  (like`self._attr_name`). See https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/entity.py#L291-L308

Reply:
I did think about that before, but the name has the county in it and isn't known until the object is initialized.

Reply:
Assign it in `__init__()` instead of `self._name = name`.